<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUser</class>
 <widget class="QDialog" name="CreateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>287</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create/Update User</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item row="5" column="0">
    <widget class="QLabel" name="passworLb">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="firmEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="passwordEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="roleBtn">
     <property name="text">
      <string>Select role</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="activaredCmbBox"/>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="locationBtn">
     <property name="text">
      <string>Select location</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="addressEdit"/>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="firmNumLb">
     <property name="text">
      <string>Firm number:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="firmLb">
     <property name="text">
      <string>Firm:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="activatedLb">
     <property name="text">
      <string>Activated:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLb">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="phoneLb">
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="addressLb">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="phoneEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="firmNumEdit"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="roleEdit"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="locationEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>firmEdit</tabstop>
  <tabstop>firmNumEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>activaredCmbBox</tabstop>
  <tabstop>roleBtn</tabstop>
  <tabstop>roleEdit</tabstop>
  <tabstop>locationBtn</tabstop>
  <tabstop>locationEdit</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
